src/components/TransferGuestDialog.vue

```diff
@@ -0,0 +1,132 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<NcDialog :name="t('guests', 'Transfer guest to new account')"
+		:out-transition="true"
+		size="small"
+		@closing="cancel">
+		<form id="transfer-guest-form"
+			@submit.prevent="submit">
+			<NcTextField ref="username"
+				:value.sync="userId"
+				:label="t('guests', 'Account name')"
```

Comment:
Maybe confusing, are we talking about user id or display name?
If it is userid, what is happening if it already used?

Reply:
This is the userId, previously it was called "Username" but recently renamed to "Account name"

Updated to show the correct error message